src/jdk.crypto.cryptoki/share/classes/sun/security/pkcs11/P11Key.java

```diff
@@ -358,6 +358,24 @@ static SecretKey secretKey(Session session, long keyID, String algorithm,
                     new CK_ATTRIBUTE(CKA_SENSITIVE),
                     new CK_ATTRIBUTE(CKA_EXTRACTABLE),
         });
+
+        if ((SunPKCS11.mysunpkcs11 != null) && (!SunPKCS11.isExportWrapKey)
```

Comment:
Please remove the redundant parentheses in `(!SunPKCS11.isExportWrapKey)` which becomes `!SunPKCS11.isExportWrapKey.get()`.

Reply:
Updated.

Reply:
The parentheses are still present.

Reply:
Removed.